### ISSUE TITLE
no part and type in response

### ISSUE THREAD

**Issue by shiftux** [IvanShamatov/sinatra-soap#9]
hello, 

i'm using this to expose a SOAP api:
```
  soap :list_incidents,
    in: {list_incidents: {productname: :string }},
    out: nil do
      destination_cmdb = cmdb_hash[params[:productname]]
      if destination_cmdb.blank?
        {error: "Error: no valid productname specified!"}
      else
        SoapApi.get_token
        response = SoapApi.get_incidents(destination_cmdb)
        {response: response}
      end
  end
```
where `response` is a string.

the wsdl i get looks like: 
```
<message name="list_cis">
<part name="list_cis" type="tns:list_cis"/></message>
<message name="list_cisResponse"></message>
```

but for the client to understand it it shoul look like this: 
```
<message name="new_incident_response">
<part name="value" type="xsd:string"/></message>
<message name="list_incident"> </message>
```
with a part_name and a type that makes sense... with wash_out this kinda used to just work...
how can i get it there?

thank you so much!

**Comment by IvanShamatov** [comment]
Hello, 
I'm really surprised you got some response at all. I mean I've dropped working on this gem and don't even remember if it works at all. Sorry, you should build it yourself or use wash_out (which is by the way great gem, though we have troubles every freaking case).

**Comment by shiftux** [comment]
haha, that was a kind of unexpected response... :)
but thaks for the honesty i guess! :) 
does wash_out work on sinatra?

**Comment by IvanShamatov** [comment]
Nope, wash_out tightly coupled with rails itself, so it wont work without rails.

**Comment by shiftux** [comment]
yea that's what i figured...

really no idea how to specify the type of the out part?!
the in part works fine, like so:
`in: {list_cis: {productname: :string }}`
it's just the out part, that i can't format to have a type...
